docs/dynamic-programming/Best time to buy and sell stocks.md

```diff
@@ -0,0 +1,365 @@
+# 📊 Best Time to Buy and Sell Stock Problems (1-5) - Dynamic Programming Solutions
```

Comment:
We need on top

```md
---
id: ___
title: ___
sidebar_label: ___
sidebar_position: ___
description: ___
tags: [___, __, ]
---
```

### Detailed Guidelines

- id: Unique identifier for the route.
- title: Title of the problem.
- sidebar_label: Problem number or label for the sidebar.
- tags: Relevant tags for the problem.
- description: A short SEO-friendly description. No HTML tags.

Reply:
Ok I will look into it.